staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -165,6 +165,8 @@ type VirtualMachineInstanceSpec struct {
 	// +listType=atomic
 	// +optional
 	AccessCredentials []AccessCredential `json:"accessCredentials,omitempty"`
+	// +optional
+	VirtualMachineRevisionName string `json:"virtualMachineRevisionName,omitempty"`
```

Comment:
As this field is optional, I think it should be a pointer value (also for compatibility as it is newly introduced).

Reply:
you can see the other optional string variables which they are not pointers

Reply:
yeah probably better to follow precedent